airbrake/notifier.py

```diff
@@ -83,6 +70,30 @@ def __init__(self, project_id=None, api_key=None, environment=None,
                 "by passing in the arguments explicitly."
             )
 
+        environment = config.get("environment",
+                                 os.getenv('AIRBRAKE_ENVIRONMENT'))
```

Comment:
Great idea, I've added an issue: https://github.com/airbrake/airbrake-python/issues/55.

This PR is a bit of a blocker, as it's important to shift to using **kwargs in the main init args to add more features. I don't think implies anything more in this particular PR right?